package.json

```diff
@@ -104,16 +103,15 @@
     "jest-junit": "^12.0.0",
     "json-loader": "^0.5.7",
     "mockdate": "^3.0.5",
-    "monaco-editor-webpack-plugin": "^4.0.0",
-    "monaco-vscode-textmate-theme-converter": "^0.1.1",
+    "monaco-editor-webpack-plugin": "^7.0.1",
+    "monaco-vscode-textmate-theme-converter": "^0.1.7",
     "sass": "~1.32.8",
     "sass-loader": "^10.2.0",
     "serve": "^11.3.2",
     "shx": "^0.3.3",
     "standard-version": "^9.3.0",
-    "string-replace-loader": "^3.0.2",
     "threads-plugin": "^1.4.0",
-    "ts-jest": "^27.0.3",
+    "ts-jest": "^26.5.6",
```

Comment:
regression?

Reply:
Had to go back as npm couldn't correctly resolve the dependencies without using "legacy" mode... which is BAD and should be avoided at all costs!!